src/main/java/pl/psnc/dei/model/CurrentUserRecordSelection.java

```diff
@@ -47,10 +48,18 @@ public boolean isRecordSelected(String recordId) {
         return selectedRecordIds.contains(recordId);
     }
 
+    public boolean isRecordSelectedForImport(String recordId) {
+        return selectedRecordIdsForImport.contains(recordId);
+    }
+
     public List<String> getSelectedRecordIds() {
         return selectedRecordIds;
     }
 
+    public List<String> getSelectedRecordIdsForImport() {
```

Comment:
Maybe Collections.unmodifiableList(selectedRecordIdsForImport) to make sure that nobody changes original list in other component?